boards/arm64/xenvm/xenvm_defconfig

```diff
@@ -18,3 +18,4 @@ CONFIG_LOG_MODE_MINIMAL=n
 CONFIG_USERSPACE=n
 
 CONFIG_TIMER_READS_ITS_FREQUENCY_AT_RUNTIME=y
+CONFIG_XEN_REGIONS=n
```

Comment:
Help please - what should be the value for extended region? 

Right now the xenvm build is broken due to ext regions issue, so it has to be fixed within "xenvm" platform instead of adding a w/a fix in each sample/test.

Reply:
Oh, I double-checked this config and now I see what is the problem. Okay, let this as is.
We just did not use our Zephyr for building xenvm (since it is not supported/used in any product), only mainline. Will be removed altogether with proper fix.

Reply:
fixed